### DISCUSSION TITLE
Add support for Gravatar images

### DISCUSSION THREAD

**Discussion by unDemian** [documenso/documenso#1367] [Ideas]
Hi there, I'm a developer with [Gravatar](https://gravatar.com/) which is known for offering a centralised avatar service that allows users to assign an avatar image to their email address, it is currently used by the likes of WordPress, GitHub or Slack with a user base in the hundreds of millions.

I haven't seen an avatar integration for Recipients and I was wondering if that is something you would be interested in, we can help with creating PRs.

On second thought, it could even be used for a new avatar field that folks might want to add to their documents. Let me know what you think.

Best!

**Comment by ElTimuro** [comment]
Hey, I always liked the concept of Gravatar. Here are a few quick thoughts:

- The recipients currently don't have an image by design to act as a traffic light for signing status
- The user account avatar would work nicely with Gravatar IMO
- It should be optional to active Gravatar though
- Using Gravatar as social prove would be an interesting angel for profiles: https://docs.documenso.com/users/profile#share-the-link
